README.md

```diff
@@ -5,7 +5,7 @@
 <!--- BADGES: START --->
 
 <!---
-![Python version](https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9-blue.svg)
+![Python version](https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue.svg)
```

Comment:
Will need a rebase which should throw up another merge conflict as this has been automated on `master`.

Reply:
This is because I screwed up. The above two lines are fenced inside comments, so the above change did nothing. 

I actually updated the correct line (line 14) to use the automated versioning yesterday (https://github.com/SeldonIO/alibi/commit/5efc04b4963a39082e73f3b3a091133162494d0e), so there is no need to update this manually at all now...

Will delete the confusing commented lines in `master` and rebase.